auto_install/install.sh

```diff
@@ -2556,7 +2734,7 @@ installScripts(){
 	else
 		# Check if bash_completion scripts dir exists and creates it if not
 		if [ ! -d "/etc/bash_completion.d" ]; then
-				mkdir -p /etc/bash_copletion.d
+				mkdir -p /etc/bash_completion.d
```

Comment:
That type was in the master branch but never surfaced, anyway if you use `mkdir -p` you don't need if. Definitely though `${SUDO}` is needed